<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="Menu" native="true">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>300</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="Iniciar_Partida">
       <property name="font">
        <font>
         <family>System</family>
        </font>
       </property>
       <property name="text">
        <string>Iniciar Partida</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Registros">
       <property name="font">
        <font>
         <family>System</family>
        </font>
       </property>
       <property name="text">
        <string>Mejor Puntaje Registrado</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Creditos">
       <property name="font">
        <font>
         <family>System</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Creditos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Salir">
       <property name="font">
        <font>
         <family>System</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="Menu_2" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="Reiniciar_Nivel">
       <property name="font">
        <font>
         <family>System</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Reiniciar Nivel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Salir_2">
       <property name="font">
        <font>
         <family>System</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Aviso_Muerte">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;MUERTO&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>Menu_2</zorder>
   <zorder>Menu</zorder>
   <zorder>Aviso_Muerte</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
